defaults/01_redhat_operators.cr.yaml

```diff
@@ -32,4 +32,7 @@ spec:
       operator: "Exists"
       effect: "NoExecute"
       tolerationSeconds: 120
-    memoryTarget: 3Mi
+    memoryTarget: 30Mi
+    extractContent:
+      cacheDir: /tmp/cache
+      catalogDir: /configs
```

Comment:
To others reviewing this PR, you may find [this comment](https://github.com/operator-framework/api/blob/5efe1a23d8c21e51c6a63a472a0d5320342187ec/pkg/operators/v1alpha1/catalogsource_types.go#L145-L158) useful.